addons/web/static/src/search/search_model.js

```diff
@@ -427,7 +427,7 @@ export class SearchModel extends EventBus {
         if (!this._context) {
             this._context = this._getContext();
         }
-        return this._context;
+        return JSON.parse(JSON.stringify(this._context));
```

Comment:
there is a deepCopy function in utils/object

Reply:
:+1: